RecoTracker/MkFit/plugins/MkFitOutputConverter.cc

```diff
@@ -246,6 +244,9 @@ TrackCandidateCollection MkFitOutputConverter::convertCandidates(const MkFitOutp
       }
     }
 
+    // Can happen with backward-search + seed-region rebuilding.
+    if (recHits.empty()) continue;
```

Comment:
@makortel  please check if this makes sense -- it was possible to happen for aggressive seed-region rebuilding and tracks where we never added any hits beyond seeds. With the new implementation this should not happen anymore -- so I can remove this check if you prefer.

Reply:
Doesn't the `Track::nTotalHits()` include the seed hits too?

Technically the protection is correct, if `recHits` is empty, bad things will happen. But having zero-valid-hit tracks at this point feels a bit strange.

Reply:
Well, what was happening was that I removed all seed hits and all that was left was some -1 and -3 hits, so recHits vector was empty and the next line was bombing.

If layer plans and iterator limits are set correctly, this should surely not happen. Perhaps a crash is better in this case, I don't know -- that's why I asked what you think :)

Reply:
Was the removing the seed hits just a test? I think the behavior here is mainly dictated by the circumstances we expect this condition to occur. E.g. if it is "never, and it occurring would imply problem elsewhere", crash or exception would be appropriate. On the other hand, for "never, occurring doesn't matter", ignoring them silently would be ok.

Adding @slava77 since the question is relevant for `reco` too.

Reply:
If things are properly configured, I'd say it should never happen.

Reply:
I removed the check.